lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -371,13 +371,9 @@ def merchant_initiated_unscheduled(options)
         return 'Y' if options.dig(:stored_credential, :initiator) == 'merchant' && options.dig(:stored_credential, :reason_type) == 'unscheduled' || options.dig(:stored_credential, :reason_type) == 'recurring'
       end
 
-      def recurring_transaction(xml, options)
-        init_txn = options.dig(:stored_credential, :initial_transaction)
-        if init_txn == true
-          xml.ssl_add_token     'Y'
-        else
-          xml.ssl_add_token     'N'
-        end
+      def add_recurring_token(options)
```

Comment:
It looks like we don't actually call this method as part of `add_auth_purchase_params` -- is this intentional?

Reply:
Ah no, that was just missed when I was cleaning up (didn't get added before pushing oops) corrected version up now.,